Easydict/NewApp/Utility/Extensions/QueryService+ConfigurableService/CustomOpenAIService+ConfigurableService.swift

```diff
@@ -149,7 +145,11 @@ private class CustomOpenAIViewModel: ObservableObject {
     }
 
     private func serviceConfigChanged() {
-        let notification = Notification(name: .serviceHasUpdated, object: nil, userInfo: nil)
+        let userInfo: [String: Any] = [
+            EZWindowTypeKey: service.windowType.rawValue,
```

Comment:
The problem is back again https://github.com/tisfeng/Easydict/pull/433#discussion_r1518580949

Reply:
Yes, just realized it will have issues when different query windows shown. I will update this part.